src/components/Badge/tests/Badge.test.tsx

```diff
@@ -13,7 +13,6 @@ describe('<Badge />', () => {
   it('accepts a status prop and renders a visually hidden label', () => {
     Object.keys(STATUS_LABELS).forEach((key: Status) => {
       const badge = mountWithAppProvider(<Badge status={STATUS_LABELS[key]} />);
-      expect(badge.prop('status')).toBe(STATUS_LABELS[key]);
```

Comment:
We shouldn't expect on the base components props since it doesn't provide value

Reply:
Should this be replaced with a more conventional test?

Reply:
Probably at some point but I don't think it's urgent since the other except still provides value and asserts what it needs to